### PR TITLE
Update language.md

### PR DESCRIPTION
**Pull request by TheMuntu**

### CODE REVIEW

File: language.md [hunk 1]
```diff
@@ -6,36 +6,35 @@ The most important tool you need to have before landing is the **offline** versi
...
-- If you can connect emotionally with the language, you will have a better experience learning, find your "why"
-- Consume your favorite media in target language. Get into interesting contents ASAP, listens to contents with transcripts, listens and reading go together.
-- Make sure you understand before you memory, understanding is addictive
-- Frequency beats intensity, better to do little often than alot at once
-- Do it when your mind feel most fresh, in the beginning of the day for example. Have a dedicated learning time
-- Invest some time in finding 3, 4 resources you like but focus on one resource at a time. Don't fall into the trap of switching to another resource when one gets challenging
+- Focus on speaking first, grammar comes easier once you know how to speak. It is important but not in the beginning.
+- Avoid the trap of spending too much time on learning grammar and vocabulary, then ending up having little speaking skills like [this guy](https://www.npr.org/sections/thetwo-way/2015/07/21/424980378/winner-of-french-scrabble-title-does-not-speak-french).
```
Comment (pierrepicaud) [inline language.md:10]: Remove the comma, fracture sentence

File: language.md [hunk 2]
```diff
@@ -6,36 +6,35 @@ The most important tool you need to have before landing is the **offline** versi
...
+- If you can connect emotionally with the language, you will have a better experience learning.
+- Consume your favorite media in the target language. Get into interesting content ASAP, listen to content with transcripts. Listening and reading go together.
+- Make sure you understand before you memorize, understanding is addictive.
+- Frequency beats intensity: It is better to do little often than a lot at once.
+- Do it when your feel the most comfortable, like in the beginning of the day for example. Have a dedicated learning time.
```
Comment (pierrepicaud) [inline language.md:15]: I want to be precise and mean "fresh", it's not the same as comfortable

File: language.md (hunk 1 above)
Comment (pierrepicaud) [reply language.md:10]: At "..., then ending up", rewrite if you feel needed

Comment (TheMuntu) [reply language.md:10]: ok, let's keep it that way!

File: language.md (hunk 2 above)
Comment (TheMuntu) [reply language.md:15]: I didn't understand it that way! It's ok!

**Pull request merged**